jbpm-persistence-jpa/src/main/java/org/jbpm/persistence/settings/JpaSettings.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jbpm.persistence.settings;
+
+import java.io.InputStream;
+import javax.xml.stream.XMLInputFactory;
+import javax.xml.stream.XMLStreamConstants;
+import javax.xml.stream.XMLStreamException;
+import javax.xml.stream.XMLStreamReader;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class JpaSettings {
+
+    static JpaSettings INSTANCE = null;
+
+    public static JpaSettings get() {
+        if (INSTANCE == null) {
+            INSTANCE = new JpaSettings();
```

Comment:
https://en.wikipedia.org/wiki/Initialization-on-demand_holder_idiom? 

And leave the field non-final to allow test changes, if necessary.